roles/kubernetes/node/templates/kubelet.env.v1beta1.j2

```diff
@@ -38,7 +38,7 @@ KUBELET_ARGS="{{ kubelet_args_base }} {% if node_taints|default([]) %}--register
 {% if kubelet_flexvolumes_plugins_dir is defined %}
 KUBELET_VOLUME_PLUGIN="--volume-plugin-dir={{ kubelet_flexvolumes_plugins_dir }}"
 {% endif %}
-{% if kube_network_plugin is defined and kube_network_plugin in ["calico", "canal", "cni", "flannel", "weave", "cilium", "kube-ovn", "kube-router", "macvlan"] %}
+{% if kube_network_plugin is defined and kube_network_plugin in ["calico", "canal", "cni", "flannel", "weave", "cilium", "kube-ovn", "kube-router", "macvlan"] and kube_version is version('v1.24.0', '<') %}
```

Comment:
I'm wondering if this was in place only for docker and the conditional was wrong, note that the kube_network_plugin is actually the full list of our supported CNIs so it is kind of pointless to check it. Maybe the right thing to do is just drop this and the subsequent line and change the conditional following it.

Reply:
Hi @cristicalin , just to make sure that I'm following this, can you make a GitHub code suggestion here?

Reply:
See the changes in https://github.com/kubernetes-sigs/kubespray/pull/8935 because I had to modify this to work on that support.